client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/SubmissionEditStepForm.jsx

```diff
@@ -293,7 +294,16 @@ const SubmissionEditStepForm = (props) => {
     const editable = !attempting && graderView;
     const visible = editable || published;
 
-    return visible ? <QuestionGrade editable={editable} id={id} /> : null;
+    return visible ? (
+      <QuestionGrade
+        editable={editable}
+        explanations={explanations}
+        gradeIsSaved={gradeIsSaved}
+        gradeIsUnsavedMessage={intl.formatMessage(translations.gradeUnsaved)}
```

Comment:
Any reason why this message has to be configurable? Looks like it's pretty standard. `QuestionGrade` could import it itself from `translations.ts`.